opentelemetry-api/tests/test_backend.py

```diff
@@ -0,0 +1,67 @@
+from importlib import reload
+import sys
+
+import pytest
+
+from opentelemetry import backend
+from opentelemetry.trace import Tracer
+
+class DummyTracer(Tracer):
+    pass
+
+DUMMY_TRACER = None
+
+def get_opentelemetry_backend_impl(type_):
+    global DUMMY_TRACER #pylint:disable=global-statement
+    assert type_ is Tracer
+    DUMMY_TRACER = DummyTracer()
+    return DUMMY_TRACER
+
+#pylint:disable=redefined-outer-name,protected-access,unidiomatic-typecheck
+
+def test_get_default(backend=backend):
+    backend = reload(backend)
+    backend._UNIT_TEST_IGNORE_ENV = True
+    tracer = backend.tracer()
+    assert type(tracer) is Tracer
```

Comment:
[`assertIsInstance`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsInstance) might save you a pylint exclusion here, unless there's a reason you need the actual `type` call.

Reply:
Here, I want to specifically assert that `tracer` is not a `DummyTracer` (but a `DummyTracer` is an instance of `Tracer`).